tests/unit/topology/allocator_binder_linear.hpp

```diff
@@ -0,0 +1,83 @@
+#ifndef ALLOCATOR_BINDER_LINEAR_HPP
+#define ALLOCATOR_BINDER_LINEAR_HPP
+
+#include <hpx/parallel/util/numa_binding_allocator.hpp>
+
+// ------------------------------------------------------------------------
+// Example of an allocator binder for 1D arrays.
+// This simply binds pages according to a linear sequence
+// with increasing numa domain number modulo the total domains available
+// ------------------------------------------------------------------------
+template <typename T>
+struct linear_numa_binder : hpx::compute::host::numa_binding_helper<T>
+{
+    linear_numa_binder(std::size_t num_pages) :
+        hpx::compute::host::numa_binding_helper<T>()
+    {
+        const std::size_t CACHE_LINE_SIZE = sysconf(_SC_LEVEL1_DCACHE_LINESIZE);
```

Comment:
`sysconf` is a Linux-specific function call. This should be hidden on other platforms.

Reply:
I have added a function to topology to get the page size on startup (mostly platform independent) and store it in a static var that is then returned by calls to hpx::threads::get_memory_page_size() - this should resolve this problem.